packages/core/selectors/getRoots.js

```diff
@@ -10,7 +10,7 @@ const getRoots = createSelector(
   [
     get('components.defaults'),
   ],
-  (defaults) => defaults.map((component) => component.name)
+  (defaults) => toArray(defaults).map((component) => component.name)
```

Comment:
An empty state returns an error. Where it could return an empty array since an error would likely break the page.